website/siteConfig.js

```diff
@@ -74,13 +74,18 @@ const siteConfig = {
   // For top-level user or org sites, the organization is still the same.
   // e.g., for the https://JoelMarcey.github.io site, it would be set like...
   //   organizationName: 'JoelMarcey'
-
+  algolia: {
+    apiKey: '522665321749697a7b612bb54dbdb0b4',
```

Comment:
I assume this is not sensitive?

Reply:
double checking with them, I don't think it is.

Reply:
Confirmed, it's meant to be public with Algolia.

Reply:
https://docsearch.algolia.com/docs/faq#can-i-share-the-apikey-in-my-repo